sqlalchemy_hana/dialect.py

```diff
@@ -540,7 +545,7 @@ def get_unique_constraints(self, connection, table_name, schema=None, **kwargs):
                     # Constraint has user-defined name
                     constraint['name'] = self.normalize_name(constraint_name)
                     constraint['duplicates_index'] = self.normalize_name(constraint_name)
-                constraints.append(constraint)
+                    constraints.append(constraint)
```

Comment:
I assume that change isn't intentional. Could you please revert that modification?